main.tf

```diff
@@ -59,17 +62,31 @@ resource "aws_db_instance" "default" {
   final_snapshot_identifier   = "${length(var.final_snapshot_identifier) > 0 ? var.final_snapshot_identifier : module.final_snapshot_label.id}"
 }
 
+resource "random_string" "default" {
+  length  = 20
```

Comment:
count     = "${var.enabled == "true" ? 1 : 0}"

let's add `count` to this resource as well since we have it for all other resources to enable/disable creation

Reply:
Added count